### DISCUSSION TITLE
How to use GetContentItemMetadataAsync to get the content type metadata?

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#8420] [Q&A]
I created a content type call "MyContent", in which has a multi-text filed. How to use code to query the metadata to get the lables and values of that field?

GetContentItemMetadataAsync  has a parameter call **IContent**, but not sure how to pass in that object with my content type name "MyContent"?

Many thanks!

**Comment by Skrypt** [comment]
You need to use the IContentDefinitionService.

**Reply by orchardc** [reply]
Thanks @Skrypt , do you have an example using IContentDefinitionService to query a content type ?
